### PR TITLE
Add test code to nugu-login-kit

### PR DESCRIPTION
**Pull request by tommy5oo**

### CODE REVIEW

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuth.kt [hunk 1]
```diff
@@ -119,12 +120,13 @@ class NuguOAuth(private val OAuthServerUrl: String?) : NuguOAuthInterface, AuthD
...
     private lateinit var options: NuguOAuthOptions
 
-    private var onceLoginListener: OnceLoginListener? = null
+    @VisibleForTesting
+    internal var onceLoginListener: OnceLoginListener? = null
 
-    // CSRF protection
-    private var clientState: String = ""
+    @VisibleForTesting
+    var clientState: String = "" // CSRF protection
```
Comment (Yuseon-Han) [inline nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuth.kt:127]: 'internal' will be little more ristrict than 'public'

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt [hunk 2]
```diff
@@ -182,7 +184,8 @@ class NuguOAuthClient(private val delegate: UrlDelegate) {
      * Handle the [AuthFlowState.STARTING] AuthFlowState for Initial status.
      * @return [AuthFlowState.REQUEST_ISSUE_TOKEN]
      */
-    private fun handleStarting(): AuthFlowState {
+    @VisibleForTesting
+    fun handleStarting(): AuthFlowState {
```
Comment (Yuseon-Han) [inline nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt:188]: In this class, 'internal' is better for ristrict App access.

**Comment by Yuseon-Han** [comment]
public methods and variables seem  better to be 'internal' for restrict to access.

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuth.kt (hunk 1 above)
Comment (tommy5oo) [reply nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuth.kt]: Applied review, thanks

File: nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt (hunk 2 above)
Comment (tommy5oo) [reply nugu-login-kit/src/main/java/com/skt/nugu/sdk/platform/android/login/auth/NuguOAuthClient.kt]: Applied review, thanks

**Review by shshshl09 — APPROVED** [review]

**Review by Yuseon-Han — APPROVED** [review]

**Pull request merged**